README.md

```diff
@@ -13,18 +13,27 @@ The SPIFFE Helper is a simple utility for fetching X.509 SVID certificates from
 
 If `-config` is not specified, the default value `helper.conf` is assumed. 
 
-The flag `-exitWhenReady` is also supported.
+CLI options:
+
+ | Flag name                       | Description                                                         |
+ |---------------------------------|---------------------------------------------------------------------|
+ | `-config`                       | Path to the configuration file                                      |
+ | `-help`                         | Print interactive help                                              |
+ | `-daemon-mode={true|false}`     | Overrides `daemon_mode` in the config file.                         |
+ | `-exitWhenReady`                | Obsolete and ignored. Use `-daemon-mode=false` instead.             |
```

Comment:
We should probably just delete this rather than documenting it.

Reply:
PR to delete the Config field: https://github.com/spiffe/spiffe-helper/pull/248